base/applications/rapps/gui.cpp

```diff
@@ -1469,6 +1469,19 @@ class CMainWindow :
         }
     }
 
+    VOID ShowAboutDlg()
+    {
+        ATL::CStringW szApp;
+        ATL::CStringW szAuthors;
+        HICON hIcon;
+
+        szApp.LoadStringW(IDS_APPTITLE);
+        szAuthors.LoadStringW(IDS_APP_AUTHORS);
+        hIcon = (HICON) LoadIconW(hInst, MAKEINTRESOURCEW(IDI_MAIN));
```

Comment:
Does it make sense to cast the function as `HICON` where such function returns a handle to a icon?

Reply:
ok